### PR TITLE
Finished bubble sort and bubble sort by functions

### PR DESCRIPTION
**Pull request by edlingao**

### CODE REVIEW

File: BubbleSortBy.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+def bubble_sort_by (arr, yield_true = false)
+    len = arr.length - 1
+    left_or_right = yield(arr[0], arr[1]) if yield_true
+    
+    loop do
+        condition = false
+        len.times{
+            |i|
+            if(!yield_true ||left_or_right < 1 )
```
Comment (inhaq) [inline BubbleSortBy.rb]: you can check block with `block_given?` kindly refactor accordingly

**Review by inhaq — CHANGES_REQUESTED** [review]
Good job on the first ruby project. Kindly add linter to your project and refactor code by the following comment. 
Once updated kindly resubmit for review.

**Comment by edlingao** [comment]
Commited every change, succefully refactor

**Review by inhaq — APPROVED** [review]
It's 👍🏻go ahead and submit your project. 
Good luck with the next project.

**Pull request merged**